Auxiliary/DirectXTexHelper.cpp

```diff
@@ -0,0 +1,58 @@
+//--------------------------------------------------------------------------------------
```

Comment:
I know that it would result in some code duplication, but I recommend eliminating this file and just pasting it into an anonymous namespace in both the JPEG and PNG modules. I really want to keep the individual modules in Auxiliary as independent and simple as possible.

Reply:
I will remove it